docs/usage.md

```diff
@@ -325,13 +325,15 @@ Loading variables can be useful in a few scenarios:
 
 ### CF-encoded time variables
 
-Notice that the `time` variable that was loaded above does not have the expected dtype. To correctly decode time variables according to the CF conventions (like `xr.open_dataset` does by default), you need to include them in an additional keyword argument `cftime_variables`:
+To correctly decode time variables according to the CF conventions, you need to pass `time` to `loadable_variables` and ensure the `decode_times` argument of `open_virtual_dataset` is set to True ( like the default behavior of `xr.open_dataset`, `decode_times` defaults to `True`).
```

Comment:
If that's the default behaviour of xarray.open_dataset then we should make it the default behaviour of open_virtual_dataset too!

Reply:
Looks like I was wrong! 
```decode_times ([bool](https://docs.python.org/3/library/functions.html#bool) or [dict-like](https://docs.python.org/3/glossary.html#term-mapping), optional) – If True, decode times encoded in the standard NetCDF datetime format into datetime objects.``` In this case, do you think we should we default to True or False?

Reply:
We should default to whatever Xarray defaults to.

Reply:
(Because once we build the VirtualiZarr Xarray backend, the kwargs will be one and the same.)

Reply:
(Because once we build the VirtualiZarr Xarray backend, the kwargs will be one and the same.)